spacewalk/admin/spacewalk-startup-helper

```diff
@@ -5,13 +5,17 @@ if [ -x "/usr/bin/lsof" ]; then
     LSOF="/usr/bin/lsof"
 fi
 
-wait_for_database() {
+check_database() {
     RETRIES=10;
     while [ $RETRIES -gt 0 ]; do
 
-        echo "select 0 from dual;" | spacewalk-sql --select-mode - &> /dev/null
+        IFS="." read -ra VARR <<< $(echo "show server_version;" | spacewalk-sql --select-mode - | sed -n 3p | xargs)
 
```

Comment:
Why not simply
g36:~ # psql --version
psql (PostgreSQL) 10.10

Reply:
In case of remote DB I want to read it from the real db we connect to.
You know we have one customer with that exception.